assemblies/assembly_dmn-models/main.adoc

```diff
@@ -12,7 +12,7 @@ include::_artifacts/author-group.adoc[]
 // Purpose statement for the assembly
```

Comment:
Regarding image `dmn-custom-data-type-add.png`: Suggest showing the actual data type that  you are creating per the instructions, as before. Meaning, **Credit_Score_Rating** data type as a **string**, instead of nothing shown. Also, did you verify the instructions for this section against the latest data type changes? Stills says "Click Add" for example to add data type, but isn't it now changed to "Click New Data Type"? From what I see in the screenshot. Suggest just walk through all the data type instruction sections to double-check it's up to date.

Reply:
@sterobin thanks for providing the inputs, have made changes as suggested, please review.